### PR TITLE
#305705666 - What is Hack Kosice Section

### PR DESCRIPTION
**Pull request by dmatis2**
This PR contains What is Hack Kosice Section with fixed background

Monday: https://hackkosice.monday.com/boards/227660128/pulses/305705666

### CODE REVIEW

File: src/organisms/home/WhatIsHK.vue [hunk 1]
```diff
@@ -1,7 +1,68 @@
+<style lang="scss" scoped>
```
Comment (MatusMak) [inline src/organisms/home/WhatIsHK.vue]: Can you move style to the bottom of the page?

File: src/organisms/home/WhatIsHK.vue [hunk 2]
```diff
@@ -1,7 +1,68 @@
...
+}
+</style>
+
 <template>
-  <div>
+  <div id="what-is-hk-background">
     <div class="container">
-      what is hK?
+      <PageHeading :display="5">What is HK?</PageHeading>
+      <SubPageHeading
```
Comment (MatusMak) [inline src/organisms/home/WhatIsHK.vue:5]: Starting and end tag are formatted a little bit weird, shouldn't `>` be on the same line as tag?

File: src/organisms/home/WhatIsHK.vue [hunk 3]
```diff
@@ -1,7 +1,68 @@
...
+        >A hackathon is a focused 24-hour event where students come together in
+        small teams to share ideas and build something new and exciting,
+        alongside a whole bunch of like-minded people. You learn immensely with
+        the help of mentors, interact with companies and have
+        fun!</SubPageHeading
+      >
+      <SubPageHeading
```
Comment (MatusMak) [inline src/organisms/home/WhatIsHK.vue:12]: Same as above

File: src/organisms/home/WhatIsHK.vue [hunk 4]
```diff
@@ -1,7 +1,68 @@
...
+        was an unforgettable experience.
+      </SubPageHeading>
+      <SubPageHeading>Let’s double the fun in 2020!</SubPageHeading>
+      <div class="text-center mt-5 pb-5">
+        <HKButton
+          link="about"
+          v-smooth-scroll="{ offset: offset }"
+          :fontSize="1.25"
+          :large="true"
+          >Learn more</HKButton
```
Comment (MatusMak) [inline src/organisms/home/WhatIsHK.vue:21]: Same as above

File: src/organisms/home/WhatIsHK.vue [hunk 5]
```diff
@@ -1,7 +1,68 @@
...
+        >Hack Kosice 2019 was the first MLH hackathon in Slovakia and it brought
+        together creative and inspiring people from over 30 countries. Along
+        with great mentors, speakers, fun mini-games and a lot of tasty food, it
+        was an unforgettable experience.
+      </SubPageHeading>
+      <SubPageHeading>Let’s double the fun in 2020!</SubPageHeading>
+      <div class="text-center mt-5 pb-5">
+        <HKButton
+          link="about"
+          v-smooth-scroll="{ offset: offset }"
```
Comment (MatusMak) [inline src/organisms/home/WhatIsHK.vue]: Remove smooth scroll, it makes no sense to use it.

File: src/organisms/home/WhatIsHK.vue [hunk 6]
```diff
@@ -1,7 +1,68 @@
...
+export default {
+  name: "WhatIsHK",
+
+  components: {
+    PageHeading,
+    SubPageHeading,
+    HKButton
+  },
+
+  computed: {
```
Comment (MatusMak) [inline src/organisms/home/WhatIsHK.vue]: Remove offset property as it will not be used once smooth scroll directive is removed

**Review by MatusMak — CHANGES_REQUESTED** [review]

File: src/organisms/home/WhatIsHK.vue (hunk 1 above)
Comment (dmatis2) [reply src/organisms/home/WhatIsHK.vue]: Done

File: src/organisms/home/WhatIsHK.vue (hunk 5 above)
Comment (dmatis2) [reply src/organisms/home/WhatIsHK.vue]: Done

File: src/organisms/home/WhatIsHK.vue (hunk 6 above)
Comment (dmatis2) [reply src/organisms/home/WhatIsHK.vue]: Done

**Review by MatusMak — APPROVED** [review]

**Pull request merged**